pkg/plugins/golang/v3/init.go

```diff
@@ -207,6 +207,9 @@ func checkDir() error {
 			if err != nil {
 				return err
 			}
+			if info.IsDir() && info.Name() != "." && strings.HasPrefix(info.Name(), ".") {
```

Comment:
So, it would allow scaffold the project when the dir starts with (.) as well. Am I right?